### PR TITLE
Unify JACK wording, fix bug, refactoring

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Includes changes of https://github.com/jamulussoftware/jamulus/pull/1887

**Context: Fixes an issue?**

No. Refactoring only.

**Does this change need documentation? What needs to be documented and how?**

No

**Status of this Pull Request**

To be tested and reviewed. It's only a small change though. 

**What is missing until this pull request can be merged?**
Tests...

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I tested my code and it does what I want (compiled; didn't try to evoke error messages though)
- [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I've filled all the content above

### CODE REVIEW

**Comment by ann0see** [comment]
Didn't check on windows, however, I think it should work as supposed.

File: src/clientsettingsdlg.cpp [hunk 1]
```diff
@@ -164,13 +167,13 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
-                                    QString().setNum ( DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES ) +
-                                    tr ( " samples. The "
-                                         "smaller the network buffers, the lower the audio latency. But at the same time "
-                                         "the network load increases and the probability of audio dropouts also increases." ) );
+    chbEnableOPUS64->setWhatsThis (
+        "<b>" + tr ( "Enable Small Network Buffers" ) + ":</b> " +
+        QString ( tr ( "If enabled, the support for very small network audio packets is activated. Very small "
+                       "network packets are only actually used if the sound card buffer delay is smaller than %1 samples. The "
+                       "smaller the network buffers, the lower the audio latency. But at the same time "
+                       "the network load increases and the probability of audio dropouts also increases." ) )
```
Comment (pljones) [inline src/clientsettingsdlg.cpp:175]: ...as well as the increase in audio artifacts caused by the codec...

Comment (ann0see) [reply src/clientsettingsdlg.cpp:175]: Yes, but do we need to add it here? This might get too technical (it already is...)

Comment (pljones) [reply src/clientsettingsdlg.cpp:175]: The feature itself is technical in nature.  If the user isn't provided with a proper explanation of the effects, they could wonder why it reduces sound quality.  It does, substantially, particularly on bass-heavy sounds.  Unless the user is _unable_ to play without it, due to latency, it's pretty much being recommended to leave it off now, during WorldJam sound check sessions.

Comment (ann0see) [reply src/clientsettingsdlg.cpp:175]:
```suggestion
                       "the network load and the probability of audio dropouts or sound artefacts increases." ) )
```

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Rebased this PR. Ready for review.

**Comment by ann0see** [comment]
@gilgongo could you please Review?

File: src/clientsettingsdlg.cpp [hunk 2]
```diff
@@ -164,13 +167,13 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
-    chbEnableOPUS64->setWhatsThis ( "<b>" + tr ( "Enable Small Network Buffers" ) + ":</b> " +
-                                    tr ( "If enabled, the support for very small network audio packets is activated. Very small "
-                                         "network packets are only actually used if the sound card buffer delay is smaller than " ) +
-                                    QString().setNum ( DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES ) +
-                                    tr ( " samples. The "
-                                         "smaller the network buffers, the lower the audio latency. But at the same time "
-                                         "the network load increases and the probability of audio dropouts also increases." ) );
+    chbEnableOPUS64->setWhatsThis (
+        "<b>" + tr ( "Enable Small Network Buffers" ) + ":</b> " +
+        QString ( tr ( "If enabled, the support for very small network audio packets is activated. Very small "
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:172]:
```suggestion
        QString ( tr ( "Enables support for very small network audio packets. These "
```
BTW I recently expanded this wording based on some conversations on the forums. See  https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Software-Manual.md#enable-small-network-buffers

File: src/clientsettingsdlg.cpp [hunk 3]
```diff
@@ -198,7 +201,7 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                      "In this case the buffer delay setting is disabled and has to be "
                                      "changed using the sound card driver. On Windows, press the "
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:202]:
```suggestion
                                     "changed using the sound card driver. On Windows, use the "
```

File: src/clientsettingsdlg.cpp [hunk 4]
```diff
@@ -198,7 +201,7 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
                                      "ASIO Device Settings button to open the driver settings panel. On Linux, "
-                                     "use the Jack configuration tool to change the buffer size." ) +
+                                     "use the JACK configuration tool to change the buffer size." ) +
                                 "<br>" +
                                 tr ( "If no buffer size is selected and all settings are disabled, an "
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:206]:
```suggestion
                                tr ( "If no buffer size is selected and all settings are disabled, this means an "
```

File: src/clientsettingsdlg.cpp [hunk 5]
```diff
@@ -198,7 +201,7 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
                                      "unsupported buffer size is used by the driver. The application "
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:207]:
```suggestion
                                     "unsupported buffer size is in use by the driver. The application "
```

Comment (gilgongo) [inline src/clientsettingsdlg.cpp:207]: When it says "the application" does that mean the driver?

File: src/clientsettingsdlg.cpp [hunk 6]
```diff
@@ -229,12 +232,12 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                          "let you choose input or outputs of your device(s). "
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:232]:
```suggestion
                                         "lets you choose input or outputs of your device(s). "
```

File: src/clientsettingsdlg.cpp [hunk 7]
```diff
@@ -229,12 +232,12 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
                                          "More information can be found on jamulus.io." );
 
-    QString strSndCardDriverSetupTT = tr ( "Opens the driver settings. Note: " ) + APP_NAME +
-                                      tr ( " currently only supports devices supporting a sample rate of " ) +
-                                      QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) +
-                                      tr ( "Hz. "
-                                           "You will not be able to select a driver/device which doesn't. "
-                                           "For more help see jamulus.io." ) +
+    QString strSndCardDriverSetupTT = QString ( tr ( "Opens the driver settings. Note: %1 currently only supports devices "
+                                                     "supporting a sample rate of %2 Hz. "
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:236]:
```suggestion
                                                     "will a sample rate of %2 Hz. "
```

File: src/clientsettingsdlg.cpp [hunk 8]
```diff
@@ -229,12 +232,12 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
+                                                     "You will not be able to select a driver/device which doesn't. "
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:237]: That statement seems to be redundant. 
```suggestion

```

File: src/clientsettingsdlg.cpp (hunk 5 above)
Comment (ann0see) [reply src/clientsettingsdlg.cpp:207]: Probably JACK/ASIO-driver?

File: src/clientsettingsdlg.cpp (hunk 8 above)
Comment (ann0see) [reply src/clientsettingsdlg.cpp:237]: This would make it clearer. IMHO it's not redundant.

Comment (gilgongo) [reply src/clientsettingsdlg.cpp:237]: OK sure, it's only words I guess.

**Comment by ann0see** [comment]
Just rewrote the history. Could you please check if all your suggestions are ok?

File: src/clientsettingsdlg.cpp [hunk 9]
```diff
@@ -196,12 +199,12 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
                                      "ASIO Device Settings button to open the driver settings panel. On Linux, "
-                                     "use the Jack configuration tool to change the buffer size." ) +
+                                     "use the JACK configuration tool to change the buffer size." ) +
                                 "<br>" +
-                                tr ( "If no buffer size is selected and all settings are disabled, an "
-                                     "unsupported buffer size is used by the driver. The application "
+                                tr ( "If no buffer size is selected and all settings are disabled, this means an "
+                                     "unsupported buffer size is in use by driver. The application "
                                      "will still work with this setting but with restricted "
                                      "performance." ) +
```
Comment (gilgongo) [inline src/clientsettingsdlg.cpp:209]: Should we say "Jamulus will still work with this setting, but may have restricted performance." ?

File: src/clientsettingsdlg.cpp (hunk 5 above)
Comment (ann0see) [reply src/clientsettingsdlg.cpp]: Hmm. No. Probably Jamulus. Changing that...

File: src/clientsettingsdlg.cpp (hunk 9 above)
Comment (ann0see) [reply src/clientsettingsdlg.cpp]: Ok. See next commit

**Comment by ann0see** [comment]
Should not be squash and merged, I think.

**Comment by ann0see** [comment]
@gilgongo ready to merge?

**Comment by gilgongo** [comment]
OK sure.

**Pull request merged**